src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,34 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = (ManufacturerDao) injector
+                .getInstance(ManufacturerDao.class);
+        System.out.println("Test create");
+        Manufacturer ford = new Manufacturer("FORD", "USA");
+        ford = manufacturerDao.create(ford);
+        Manufacturer audi = new Manufacturer("AUDI", "Germany");
+        audi = manufacturerDao.create(audi);
+        Manufacturer lanos = new Manufacturer("LANOS", "UKRAINE");
+        lanos = manufacturerDao.create(lanos);
+        System.out.println("Test getAll");
+        manufacturerDao.getAll().forEach(System.out::println);
+        System.out.println("Test update. Changed name in lanos variable to tesla");
+        lanos.setName("tesla");
+        System.out.println(manufacturerDao.update(lanos));
```

Comment:
we can use it as business logic (not only for displaying it at console) and then sout all data from db
```suggestion
        manufacturerDao.update(lanos)
        manufacturerDao.getAll().forEach(System.out::println);
```